### ISSUE TITLE
DeprecationWarning: invalid escape sequence \c in test/plugins/pylint_test.py

### ISSUE THREAD

**Issue by cclauss** [edx/edx-lint#94]
https://travis-ci.com/cclauss/edx-lint/jobs/180003543#L247-L248
```
test/plugins/pylint_test.py::test_that_we_can_test_pylint
  <unknown>:12: DeprecationWarning: invalid escape sequence \c
```
https://github.com/edx/edx-lint/blob/master/test/plugins/pylint_test.py

**Comment by nedbat** [comment]
Yes, I'm not sure how to silence that warning.  I added the `\\c` in that docstring specifically to generate a pylint warning in that test.  If you have an idea for improvement, I'm all ears.

**Comment by cclauss** [comment]
Make it an r'string' or a b'string' ?

**Comment by nedbat** [comment]
Fixed in #95
